android-sdk/src/main/java/ai/evolv/android_sdk/Options.java

```diff
@@ -0,0 +1,183 @@
+package ai.evolv.android_sdk;
+
+import ai.evolv.android_sdk.httpclients.HttpClient;
+
+
+
+public class Options {
```

Comment:
Not clear what this class is doing - all commented out?

Reply:
was removed